iplocation/iplocation.go

```diff
@@ -0,0 +1,21 @@
+package iplocation
```

Comment:
The package that clients depend upon. It serves as an abstraction if we want to change the resolving underlying library or similar.